### ISSUE TITLE
windows: Python 3 conda install is still on 0.4.x

### ISSUE THREAD

**Issue by Francois-Bergeron** [cvxgrp/cvxpy#701]
Conda install for Python 2.7 is on 1.0.x, but Python 3 is still on 0.4
using `conda install -c cvxgrp cvxpy`

```
(base) C:\>conda search -c cvxgrp cvxpy
WARNING: The conda.compat module is deprecated and will be removed in a future release.
Loading channels: done
# Name                       Version         Build         Channel
cvxpy                          0.4.2            py_0           cvxgrp
cvxpy                         0.4.10  py36hff9b014_0  cvxgrp
cvxpy                          1.0.1  py27hf9d7886_0  cvxgrp
```

**Comment by rileyjmurray** [comment]
This is only true for Windows
```
riley@elephantmk2:~$ conda search -c cvxgrp cvxpy --platform win-64
Loading channels: done
# Name                       Version           Build  Channel             
cvxpy                          0.4.2            py_0  cvxgrp              
cvxpy                         0.4.10  py36hff9b014_0  cvxgrp              
cvxpy                          1.0.1  py27hf9d7886_0  cvxgrp 
```
Renaming the issue. We will get this fixed, but for now please use ``pip install cvxpy`` on Windows. Note that you can run "pip install" within a given conda environment; just make sure to install pip to that conda environment first. You will also need to have an appropriate C++ compiler installed (the Microsoft Visual Studio compilers will do the job).

**Comment by rileyjmurray** [comment]
Per

https://anaconda.org/cvxgrp/cvxpy/files?sort=basename&sort_order=desc

It looks like up-to-date (1.0.24) cvxpy is available for Windows (python 3.5 and 3.6). With my most recent pull request, that should also include python 3.7.
